awscli/clidriver.py

```diff
@@ -656,10 +657,15 @@ def invoke(self, service_name, operation_name, parameters, parsed_globals):
             value is returned.
 
         """
+        if hasattr(parsed_globals, 'read_timeout'):
```

Comment:
`if parsed_globals.read_timeout is None`.  `read_timeout` should always be in parsed globals since it's defined in `cli.json`.